flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/ExecNode.java

```diff
@@ -31,6 +32,18 @@
  */
 public interface ExecNode<T> {
 
+	/**
+	 * Returns a string which describes this node.
+	 * TODO rename to `getDescription` once all ExecNodes do not extend from FlinkPhysicalRel,
+	 *  because RelNode already has `getDescription` method.
+	 */
+	String getDesc();
```

Comment:
this pr should wait https://github.com/apache/flink/pull/14315, and use this method in ExecPlanDumper right?

Reply:
no, if this pr is merged before https://github.com/apache/flink/pull/14315, I will update https://github.com/apache/flink/pull/14315. and if https://github.com/apache/flink/pull/14315 is merged before this pr, I will update ExecPlanDumper